CHANGES.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+### Shadows TODO
```

Comment:
I didn't see these, but perhaps that is part of the ` Integration with higher level constructs like Viewer.` task item.